src/main/java/org/broadinstitute/hellbender/tools/exome/allelefraction/AFData.java

```diff
@@ -0,0 +1,77 @@
+package org.broadinstitute.hellbender.tools.exome.allelefraction;
+
+import org.apache.commons.lang.math.IntRange;
+import org.broadinstitute.hellbender.tools.exome.AllelicCount;
+import org.broadinstitute.hellbender.tools.exome.SegmentedModel;
+import org.broadinstitute.hellbender.tools.exome.TargetCollection;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.mcmc.DataCollection;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+/**
+ *
+ * Created by davidben on 11/10/15.
+ */
+public final class AFData implements DataCollection {
+    // allelic counts indexed by het
```

Comment:
this is fair.  by previous usage, i meant the python version---when porting stuff over i think i tended to refer to the sites as SNPs as that version did.

if you think about it, it's all a bit confusing, as there are 3 different categories of SNPs floating around: 1) those in the list of common SNPs, 2) the hets that get pulled down from the normal, and 3) the corresponding sites in the tumor, which may or may not be het (due to LOH).  so i don't have any strong opinion as to which term we use, as long as everything is consistent---and we can certainly use the convention used elsewhere in the GATK.